packages/oidc/src/token-util.ts

```diff
@@ -97,7 +97,7 @@ export function saveToken(token: VercelTokenResponse, projectId: string): void {
     }
     const tokenPath = path.join(dir, 'com.vercel.token', `${projectId}.json`);
     const tokenJson = JSON.stringify(token);
-    fs.mkdirSync(path.dirname(tokenPath), { mode: 0o660, recursive: true }); // read/write perms for owner only
+    fs.mkdirSync(path.dirname(tokenPath), { mode: 0o770, recursive: true }); // read/write/exec perms for owner/group only, x required for dir ops
```

Comment:
do we need to worry about existing directories with bad perms, like look for them and remove or edit their perms or anything?